sources/Launcher/DYNMarginCalculationLauncher.cpp

```diff
@@ -440,47 +450,45 @@ MarginCalculationLauncher::prepareEvents2Run(const task_t& requestedTask,
 }
 
 void
-MarginCalculationLauncher::launchScenario(const boost::shared_ptr<Scenario>& scenario, const std::string& baseJobsFile,
+MarginCalculationLauncher::launchScenario(const AnalysisContext& context, const boost::shared_ptr<Scenario>& scenario,
     const double variation, SimulationResult& result) {
-  if (nbThreads_ == 1)
-    std::cout << " Launch task :" << scenario->getId() << " dydFile =" << scenario->getDydFile() << std::endl;
+  std::stringstream ss;
```

Comment:
Please keep the condition on number of threads

Reply:
OK